ui/app/pages/create-account/connect-hardware/account-list.js

```diff
@@ -18,6 +19,10 @@ class AccountList extends Component {
         name: `Legacy (MEW / MyCrypto)`,
         value: mewKey,
       },
+      {
+        name: `Trezor`,
```

Comment:
Let's label this `BIP44 Standard` instead of `Trezor`

Reply:
This makes sense, I'm wondering if this might not be intuitive for many Trezor users. Would something like `BIP44 Standard (Trezor)` address both concerns?

Reply:
How about `BIP44 Standard (e.g. Trezor)`? You raise a good point about Trezor users not being familiar with BIP44, but still I'd like to avoid giving the impression that this is Trezor-specific.

Reply:
Sounds good to me.